.gitignore

```diff
@@ -16,3 +16,7 @@ docs/api/_build
 *.rej
 *~
 .*.swp
+
+# Ignore imglog files
+docs/examples/imglog/
+docs/examples/*.txt
```

Comment:
Hmmm, while it might be a good idea to ignore files, the `imglog` folder and well as others might be generated in the "current directory" which could end up in different locations depending on how you run code. So for now let's simply remove the files from your own tree and commit afterwards, no need to define files to ignore until we find a better solution.